lld/test/ELF/arm-thumb-jump8-11.test

```diff
@@ -0,0 +1,108 @@
+# RUN: split-file %s %t
+# RUN: clang -x c -E -P -DLITTLEENDIAN %t/a.yaml -o %t/a.yaml
+# RUN: clang -x c -E -P -DLITTLEENDIAN %t/b.yaml -o %t/b.yaml
+# RUN: yaml2obj %t/a.yaml -o %t/a.o
+# RUN: yaml2obj %t/b.yaml -o %t/b.o
+# RUN: ld.lld %t/a.o %t/b.o -o %t/linked.o
+# RUN: llvm-objdump -d %t/linked.o | FileCheck %s --check-prefixes=CHECK,CHECK-LE
+
+# RUN: split-file %s %t
+# RUN: clang -x c -E -P -DBIGENDIAN %t/a.yaml -o %t/a.yaml
+# RUN: clang -x c -E -P -DBIGENDIAN %t/b.yaml -o %t/b.yaml
+# RUN: yaml2obj %t/a.yaml -o %t/a.o
```

Comment:
Is there a reason to use yaml for the test?

You can get R_ARM_JMP11 and R_ARM_JMP8 out of the assembler by putting the source and destination in different sections.

```
        .thumb
        .section .text.1, "ax", %progbits
target:
        bx lr
        .section .text.2, "ax", %progbits
        b.n target     // R_ARM_THM_JUMP11
        beq.n target // R_ARM_THM_JUMP8
```

Reply:
Indeed it's alot simpler to use the assembler for the test. I got tunnel visioned with yaml2obj.

Done.